src/Auth0.php

```diff
@@ -544,6 +544,42 @@ public function setRefreshToken($refreshToken)
         return $this;
     }
 
+    /**
+     * Renews the access token and ID token using an existing refresh token.
+     * 
+     * @throws CoreException If the Auth0 object does not have access token and refresh token
```

Comment:
Also needs `@throws ApiException` for handling `$this->authentication->oauth_token` (and feedback below)